### DISCUSSION TITLE
Question regarding moov X12 library

### DISCUSSION THREAD

**Discussion by palsri2004** [moov-io/ach#1437] [General]
Hello,

I would like to use X12 moov library :https://github.com/moov-io/x12 to create EDI format and wanted to know if this library is maintained/supported by the team and updated for any EDI rule changes.

Thanks,

**Comment by adamdecaf** [comment]
We we find updates to EDI they're made in the library. Have you found some updates that need to be made? moov-io/x12 is not a complete implementation of the EDI standards currently.

**Comment by palsri2004** [comment]
Thanks @adamdecaf for your response. I didn't see any updates to EDI. We would like to use this library for our application to generate EDI format and specifically interested in 820 format for ACH. I see the release is not cut for 820 rule. Do you have any timeline for this? Additionally this library is supported for any vulnerabilities? We would like to leverage it in production. Thanks again.

**Reply by adamdecaf** [reply]
We would love to collaborate with you on improving the 820 rule set. If you have some work in progress we can help get it merged and shipped. 

We monitor all libraries and actively resolve any security or vulnerability issues. If you believe you've found a problem [please reach out](https://github.com/moov-io/ach?tab=security-ov-file#readme).

**Reply by palsri2004** [reply]
Thanks @adamdecaf . It seems our rules will be bit different in terms of what are acceptable values for segments and also number of segments may differ.  How would you like to proceed on that? Shall we fork your repository?

**Reply by adamdecaf** [reply]
Yea go ahead and fork the repositories. When you've got a working example we can look to see what it'll take to integrate back into the moov-io repositories.
